docs/layers/path-layer.md

```diff
@@ -106,6 +112,15 @@ If the color alpha (the fourth component) is not provided,
 Method called to determine the width to draw each path with.
 Unit is meters.
 
+##### `getDashArray` (Function, optional)
```

Comment:
Fine for now. For the API audit, thinking that this name could be clearer and might be better as `getDashSizeAndGap`, or maybe `getDashParameters`, it is more extensible.

Reply:
I was thinking the `stroke-dasharray` style in SVG.

Reply:
Sure. If there is good precedent, then why not. Without that context it sounded odd. 

Taking a look at the [`stroke-dasharray` style](https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/stroke-dasharray) I see that unlike ours, that style actually supports actual arrays of multiple dashes and gaps. So not 100% obvious but if our other names are consistently based on SVG styles it may be the right choice.

Reply:
True, and you can do a lot of nice tricks with that, e.g. [animating a path](https://bl.ocks.org/mbostock/5649592). We should consider future support for the full `stroke-dasharray` path as a future feature.

Reply:
> you can do a lot of nice tricks with that, e.g. animating a path.

Cool! While that animation might take some work to achieve, it struck me that a simple `gapOffset` property/uniform could allow us to easily animate a ["marching ants"](https://ideastocreations.blogspot.com/2016/06/ae-building-selection-box-shaped-by-2.html)  selection.